src/main/resources/treeformats.yml

```diff
@@ -51,6 +51,17 @@ formats:
     version:
       regex: "@([^@\\s]+)(\\s\\S+)?$"
       group: 1
+  - format: nuget
+    description: "Nuget tree output"
+    tool: "See https://github.com/Brend-Smits/NugetTree"
+    types:
+      "": "nuget"
+    name:
+      regex: "([^ ]*)"
```

Comment:
At least one "no space" is perhaps better expressed as `\\S+'.

My suggestion would be to figure out what the allowed characters are for NuGet package names.

Reply:
I do not think this is necessary anymore after changing the Regex from ``regex: "([^ ]*)"`` to ``regex: "(?<= )(\\d.+)``
NuGet package names accept characters, digits, and even some Unicode... 
Do you disagree?